src/project/qml/MuseScore/Project/ScorePropertiesDialog.qml

```diff
@@ -0,0 +1,219 @@
+
+    ScorePropertiesModel {
+        id: scorePropertiesModel
+    }
+
+    Component.onCompleted: {
+        scorePropertiesModel.load()
+    }
+
+    ColumnLayout {
+        anchors.fill: parent
+        Layout.preferredWidth: parent.width
+        spacing: 8
+
+        Column {
+            id: headerPart
+            spacing: 4
+
+            RowLayout {
+                width: root.contentWidth
+                spacing: 8
+
+                StyledTextLabel {
+                    Layout.preferredWidth: 110
+                    Layout.preferredHeight: 30
+                    text: qsTrc("notation", "File path:")
+                    font: ui.theme.bodyBoldFont
+                    horizontalAlignment: Text.left
+                }
+
+                TextInputField {
+                    Layout.fillWidth: true
+                    height: 30
+                    currentText: scorePropertiesModel.filePath
+                    readOnly: true
+                }
+
+                FlatButton {
+                    Layout.preferredWidth: 30
+                    Layout.preferredHeight: 30
+                    icon: IconCode.OPEN_FILE
+                    onClicked: scorePropertiesModel.openFileLocation()
+                }
+            }
+
+            RowLayout {
+                width: root.contentWidth
+                spacing: 8
+
+                StyledTextLabel {
+                    Layout.preferredWidth: 110
+                    text: qsTrc("notation", "MuseScore version:")
+                    font: ui.theme.bodyBoldFont
+                    horizontalAlignment: Text.left
+                }
+
+                TextInputField {
+                    Layout.fillWidth: true
+                    currentText: scorePropertiesModel.version
+                    readOnly: true
+                }
+
+                StyledTextLabel {
+                    text: qsTrc("notation", "Revision:")
+                    font: ui.theme.bodyBoldFont
+                    horizontalAlignment: Text.left
+                }
+
+                TextInputField {
+                    Layout.preferredWidth: 170
+                    currentText: scorePropertiesModel.revision
+                    readOnly: true
+                }
+
+                StyledTextLabel {
+                    text: qsTrc("notation", "API-Level:")
+                    font: ui.theme.bodyBoldFont
+                    horizontalAlignment: Text.left
+                }
+
+                TextInputField {
+                    Layout.preferredWidth: 100
+                    currentText: scorePropertiesModel.apiLevel
+                    readOnly: true
+                }
+
+                Item {                      //To mimic a flatbutton (so that it aligns with the open folder button)
+                    Layout.preferredWidth: 30
+                    Layout.preferredHeight: 30
+                }
+            }
+        }
+
+        SeparatorLine {}
+
+        StyledListView {
+            id: bodyTags
```

Comment:
Probably it should be customTags

Reply:
I named it bodyTags because it also includes tags like Composer, Copyright, Creation date which are not custom

Reply:
Not all these tags are really custom; this namely also includes standard tags like `workTitle`, `composer`, etc.
Maybe a more neutral id like `tagsListView` is better.